static/app/components/workflowEngine/form/control/priorityControl.tsx

```diff
@@ -13,43 +13,45 @@ import {IconArrow, IconChevron} from 'sentry/icons';
 import {t} from 'sentry/locale';
 import {space} from 'sentry/styles/space';
 import {PriorityLevel} from 'sentry/types/group';
+import {
+  METRIC_DETECTOR_FORM_FIELDS,
+  useMetricDetectorFormField,
+} from 'sentry/views/detectors/components/forms/metricFormData';
 
 function ThresholdPriority() {
-  const lowThresholdDirection = useFormField<string>('conditionGroup.conditions.0.type')!;
-  const lowThreshold = useFormField<string>('conditionGroup.conditions.0.comparison')!;
+  const conditionType = useMetricDetectorFormField(
+    METRIC_DETECTOR_FORM_FIELDS.conditionType
+  );
+  const conditionValue = useMetricDetectorFormField(
+    METRIC_DETECTOR_FORM_FIELDS.conditionValue
+  );
   return (
     <div>
-      {lowThresholdDirection === ''
-        ? t('Above')
-        : lowThresholdDirection === 'above'
-          ? t('Above')
-          : t('Below')}{' '}
-      {lowThreshold === '' ? '0s' : lowThreshold + 's'}
+      {conditionType === 'gt' ? t('Above') : t('Below')}{' '}
+      {conditionValue === '' ? '0s' : conditionValue + 's'}
```

Comment:
nit: prefer string interpolation

```suggestion
      {conditionValue === '' ? '0s' :  `${conditionValue}s`}
```
should this also handle `null` or `undefined` cases?